src/plugins/discover/public/components/discover_grid_flyout/discover_grid_flyout.tsx

```diff
@@ -164,14 +165,15 @@ export function DiscoverGridFlyout({
       isPlainRecord,
       onFilter,
       removeColumn,
+      flyoutCustomization?.docViewsRegistry,
     ]
   );
 
   const contentActions = useMemo(
     () => ({
-      addFilter: onFilter,
-      addColumn,
-      removeColumn,
+      filter: onFilter,
+      onAddColumn: addColumn,
+      onRemoveColumn: removeColumn,
```

Comment:
the prefix `on` sounds more like an Event Listener ? As if we want something to happen as an response to something. `onAddColumn` feels like we want some other action to happen when a column is added ?

Reply:
I agree 100%. I kept this same naming to follow the defined interface of the `DocViewRenderProps` and avoid duplicating other props with different naming, @lukasolson what's your take on this? I'm ok with applying a wider renaming of the props if necessary.

Reply:
I just reached the code below to see that DocViewer is using the same naming 😆

Reply:
No strong preference here.